### PR TITLE
[16.0][MIG] maintenance_purchase

### PR DESCRIPTION
**Pull request by IT-Ideas**

### CODE REVIEW

**Review by mamcode — APPROVED** [review]
LGTM 👍🏼  Thanks for the migration and unit tests.

**Comment by IT-Ideas** [comment]
@pedrobaeza

File: maintenance_purchase/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "name": "Maintenance Purchase",
     "summary": """
         Create Equipments with purchases""",
-    "version": "14.0.1.0.2",
+    "version": "16.0.1.0.2",
```
Comment (pedrobaeza) [inline maintenance_purchase/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Comment by IT-Ideas** [comment]
Here you are @pedrobaeza

**Review by mamcode — APPROVED** [review]

File: maintenance_purchase/tests/test_purchase_order.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
+# Copyright 2024 ACSONE SA/NV
```
Comment (pedrobaeza) [inline maintenance_purchase/tests/test_purchase_order.py:1]: Is this copyright correct? Don't you put as contributor?

**Review by pedrobaeza — APPROVED** [review]
Thanks for the tests

File: maintenance_purchase/tests/test_purchase_order.py (hunk 2 above)
Comment (IT-Ideas) [reply maintenance_purchase/tests/test_purchase_order.py:1]: Thanks @pedrobaeza,
I am currently working for Acsone, so the copyright is correct 🙂

**Comment by pedrobaeza** [comment]
Your GitHub login is a bit cryptic! While you aren't Jia Tan, it's OK, hehe

/ocabot merge nobump

**Comment by IT-Ideas** [comment]
> Your GitHub login is a bit cryptic! While you aren't Jia Tan, it's OK, hehe
> 
> /ocabot merge nobump

Yeah I know ^^.
I am a freelancer (https://it-ideas.be), that's the reason of my GitHub account name.

I worked in the R&D @ Odoo for 3 years (we had been in contact at that time, I was responsible of the project, timesheet, planning, fsm and helpdesk apps 😉), now I am working for Acsone for a year.

**Comment by pedrobaeza** [comment]
Yeah, I remember exchanging some conversations in Odoo PRs! Welcome to the outsider world. Hope to meet you as Spanish OCA days in May.

**Pull request merged**